test/e2e/backward_compat_test.go

```diff
@@ -113,10 +106,27 @@ var _ = Describe("When upgrading AAD Pod Identity", func() {
 			IdentityClientID: identityClientID,
 		})
 
+		By("Deleting the ConfigMap used to store upgrade information")
```

Comment:
Moving the configmap deletion to after to ensure there is no configmap right before the upgrade to the latest version